test/test-network/systemd-networkd-tests.py

```diff
@@ -502,6 +502,7 @@ def test_reconfigure(self):
         check_output('ip address del 10.2.2.4/16 dev dummy98')
 
         check_output(*networkctl_cmd, 'reconfigure', 'dummy98', env=env)
+        time.sleep(1)
```

Comment:
hmm, aren't we updating the link state file synchronously before returning in the bus call?